index.html

```diff
@@ -289,6 +290,37 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <section class="contact flex" id="contact">
+        <p class="parag text-dark">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          class="flex"
+          action="https://formspree.io/f/xdobralq"
+          method="post"
+          id="mail"
+        >
+          <input
```

Comment:
- [ ] [OPTIONAL] This is an optional change since the previous reviewer did not mention it but I would suggest you add it as it may be part of the same resubmission and a contribution to your professionalism when creating Html forms, please add the `required` flag to all the input elements in order to trigger the validation on form submit, good trick, right? yes 🙃

- [ ] As mentioned by the previous reviewer, referring to this [requirement,](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) we are required to specify the max length of characters the input should accept, kindly add it and resubmit your PR.